starfish/filters.py

```diff
@@ -38,7 +11,7 @@ def gaussian_high_pass(img, sigma):
     :param sigma: Standard deviation of gaussian kernel :type int
     :return: Filtered image, same shape as input :type ndarray
     """
-    blurred = gaussian_low_pass(img, sigma)
+    blurred = GaussianLowPass.filter(img, sigma)
```

Comment:
I believe you want to make it:

`GaussianLowPass(sigma).filter(img)`

Reply:
Yep, that's consistent with our conversation. Will fix.